app/assets/stylesheets/admin.css.scss

```diff
@@ -47,8 +47,25 @@ ul.inline li {
       font-weight: 600;
       opacity: 1;
     }
+    &:hover {
+      color: #293f54 !important;
```

Comment:
I believe using !important is a bad practice, and should be avoided because it makes debugging and updating harder than it needs to be... Is there a big reason to use it? @technical-boy what do you think?

Reply:
Yes, I used !important because for some reason the styles of the buttons remain the same and doesn´t behave properly so the only solution I found was to use !important